lib/utilities/mouseDownDrag.js

```diff
@@ -1,49 +1,72 @@
 /**
+ * Callback which is called upon at some point during a drag event in React.
+ *
+ * @callback drawCallback
+ * @param {event} event The original mouse event.
+ */
+
+/**
  * This utility implements dragging for SVG elements in React. That is, it
  * wraps mouse events and simulates drag events, but does not support complex
  * drag-and-drop functionality.
- * 
+ *
  * Users of this utility should define the three callbacks and then set the
  * onMouseDown of the draggable element to mouseDownDrag.bind(this,
  * someUniqueName, myPseudoDragStartCallback, myPseudoDragEndCallback,
  * myPseudoDragCallback). someUniqueName can be anything, as long it's unique
  * within the component.
- * 
+ *
  * The using component can check its state for the key
  * (someUniqueName + '_dragging') to see if a particular draggee is dragging.
+ *
+ * @param {*} name - Name which is used to uniquely identify *this* drag event.
+ * @param {dragCallback} pseudoDragStartCallback - Callback which is fired
+ * upon the start of a drag event.
+ * @param {dragCallback} pseudoDragEndCallback - Callback which is fired upon
+ * the end of a drag event.
+ * @param {dragCallback} pseudoDragCallback - Callback which is fired when the
+ * mouse moves. Use this to update the position of whatever is being drawn.
+ * @param {event} event - The original mouse event?
+ * @returns {void}
```

Comment:
I just realised that the format is slightly incorrect. The parts in curly braces should be types, not identifiers. I will fix this in my own branch.